Formula/traefik.rb

```diff
@@ -22,7 +22,7 @@ def install
 
     cd "src/github.com/containous/traefik" do
       system "go", "generate"
-      system "go", "build", "-o", bin/"traefik", "./cmd/traefik"
+      system "go", "build", "-trimpath", "-o", bin/"traefik", "./cmd/traefik"
```

Comment:
I think these changes shouldn't be in this commit

Reply:
Sure, I will remove it.

Reply:
Not quite sure that change got in, maybe doing two things at one time 😅